src/example/Simplest.js

```diff
@@ -0,0 +1,71 @@
+import ActionManager from "../action/ActionManager.js";
+import DoubleJumpFilter from "./filters/DoubleJumpFilter.js";
+
+function main() {
```

Comment:
I know this is pseudocode, but I feel like this code should live in a doc/README. on its own, without a functional end-to-end example, it's confusing to me. I don't know the target game, inputs, etc.

Reply:
Yes, I agree that it's confusing. I wanted the nice diff and comment functionality so I put it into a PR, but I don't think we should merge this PR.